compiler/src/iree/compiler/Dialect/Stream/Transforms/test/encode_host_tensors_packing_i1_attr.mlir

```diff
@@ -0,0 +1,22 @@
+// RUN: iree-opt --split-input-file --iree-stream-encode-host-tensors %s | FileCheck %s
+
+#packed = #iree_encoding.packed_storage
+func.func @unaligned_i1_size() -> index {
```

Comment:
use util.func within the IREE codebase if outside of codegen (func.func is converted to util.func very early on)